diff -aurN aiksaurus-1.0.1.orig/configure.ac aiksaurus-1.0.1/configure.ac
--- aiksaurus-1.0.1.orig/configure.ac	2003-06-26 23:22:31.000000000 +0200
+++ aiksaurus-1.0.1/configure.ac	2004-03-11 13:43:26.972631904 +0100
@@ -121,6 +121,7 @@
 if test $aik_fe_gtk = yes; then
 	GTK_DEFS="-DG_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED -DGDK_PIXBUF_DISABLE_DEPRECATED"
 	GTK_DEFS="$GTK_DEFS -DGTK_DISABLE_DEPRECATED -DGNOME_DISABLE_DEPRECATED"
+	GTK_DEFS=""
 fi
 
 AC_SUBST(GTK_CFLAGS)
