--- aiksaurus-1.2.1/base/caiksaurus.cpp.orig	2014-01-06 13:27:56.420352733 +0100
+++ aiksaurus-1.2.1/base/caiksaurus.cpp	2014-01-06 13:29:36.108561088 +0100
@@ -254,7 +254,7 @@ static void handleArguments(int argc, ch
         /* see if we got --version. */
         if ( !strcmp(arg, "--version") || !strcmp(arg, "-v") )
         {
-            printf(version);
+            fputs(version, stdout);
             exit(0);
         }
 
@@ -262,7 +262,7 @@ static void handleArguments(int argc, ch
         else if ( !strcmp(arg, "--help") || !strcmp(arg, "-h") )
         {
             printf("Usage: %s [word]\n\n", argv[0]);
-            printf(help);
+            fputs(help, stdout);
             exit(0);
         }   
     }
@@ -271,7 +271,7 @@ static void handleArguments(int argc, ch
     if (argc != 2)
     {
         printf("Usage: %s [word]\n", argv[0]);
-        printf(help);
+        fputs(help, stdout);
         exit(1);
     }
 }
